microovn/src/charm.py

```diff
@@ -47,13 +47,10 @@ def call_microovn_command(*args, stdin=None):
 class MicroovnCharm(ops.CharmBase):
     """The implementation of the majority of the charms logic."""
 
-    _stored = ops.StoredState()
```

Comment:
This looks like a refactor unrelated to the removal of the submodule. Please separate out to own commit.